tensorflow/python/ops/math_ops.py

```diff
@@ -4104,7 +4104,8 @@ def _tensordot_axes(a, axes):
       a_free_dims = ops.convert_to_tensor(a_free_dims, dtype=dtypes.int32)
       b_free_dims = ops.convert_to_tensor(b_free_dims, dtype=dtypes.int32)
       product = array_ops.reshape(
-          ab_matmul, array_ops.concat([a_free_dims, b_free_dims], 0), name=name)
+          ab_matmul, array_ops.concat(
+          [a_free_dims, b_free_dims], 0), name=name)
```

Comment:
I'm not sure that this actually provides a more readable result?

I'd suggest reverting to the before:

```
product = array_ops.reshape(
    ab_matmul, array_ops.concat([a_free_dims, b_free_dims], 0), name=name)
```

Or to keep every parameter to the function on it's own line:

```
product = array_ops.reshape(ab_matmul,
                            array_ops.concat([a_free_dims, b_free_dims], 0),
                            name=name)
```

Reply:
Thanks @tomhennigan. The both two ways exceed 80 char in a line. Which one do you prefer to edit?

Reply:
I _think_ that the if we leave it as is (e.g. the first suggestion) it's exactly 80ch so I'd suggest leaving it as is for now:

```shell
$ echo -n "          ab_matmul, array_ops.concat([a_free_dims, b_free_dims], 0), name=name)" | wc -c
      80
```